images/ose-network-interface-bond-cni.yml

```diff
@@ -18,10 +22,11 @@ for_payload: true
 from:
   builder:
   # Binaries must be compiled for the RHCOS version of RHEL
-  - stream: rhel-9-golang
-  - stream: rhel-8-golang
+  - stream: rhel-9-golang-1.23
+  - stream: rhel-8-golang-1.23
   member: openshift-enterprise-base-rhel9
 name: openshift/ose-network-interface-bond-cni-rhel9
 payload_name: network-interface-bond-cni
 owners:
 - carlosg@redhat.com
+canonical_builders_from_upstream: true
```

Comment:
not sure if we need this since ci build root is pointing to `ci-openshift-build-root-extra.rhel9` and extra is 1.23